tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_RAP_suite_RAP.yaml

```diff
@@ -21,9 +21,6 @@ rocoto:
     metatask_run_ensemble:
       task_run_fcst_mem#mem#:
         walltime: 01:00:00
```

Comment:
@mkavulich So are these lines being removed to enable the `get_obs_...` tasks?  If so, does that mean we don't need to stage the obs for these tests?

Reply:
I simply added a line to the script that checks if all the observations already exist at the specified location. If they already exist, we do nothing, otherwise we attempt to pull them from HPSS. This removes the need for de-activating the tasks depending on where you want to get your observations.